metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -686,3 +686,95 @@ def command(ctx: click.Context) -> None:
         ctx.exit(1)
 
     return command
+
+
+def get_session_login_as(
+    username: str, password: str, frontend_url: str
+) -> requests.Session:
+    session = requests.Session()
+    headers = {
+        "Content-Type": "application/json",
+    }
+    system_auth = get_system_auth()
+    if system_auth is not None:
+        session.headers.update({"Authorization": system_auth})
+    else:
+        data = '{"username":"' + username + '", "password":"' + password + '"}'
+        response = session.post(f"{frontend_url}/logIn", headers=headers, data=data)
+        response.raise_for_status()
+    return session
+
+
+def ensure_valid_gms_url_acryl_cloud(url: str) -> str:
```

Comment:
python style thing - methods only used in this file should be prefixed with an underscore. it's a poor man's `private` keyword

```suggestion
def _ensure_valid_gms_url_acryl_cloud(url: str) -> str:
```

Reply:
done